src/scripts/main.js

```diff
@@ -1,3 +1,301 @@
 'use strict';
 
-// write code here
+document.body.style.alignItems = 'flex-start';
+
+const headList = document.querySelector('thead');
+const body = document.querySelector('tbody');
+
+let sortOrder = '';
+let sortHeader = null;
+
+function normalizeNumber(number) {
+  let newNumber;
+
+  if (number.includes('$')) {
+    newNumber = number.replace(/[$,]/g, '');
+  } else {
+    newNumber = number;
+  }
+
+  return newNumber;
+};
+
+headList.addEventListener('click', (e) => {
+  const toSort = e.target.cellIndex;
+  const target = e.target;
+
+  if (sortHeader !== target) {
+    sortOrder = 'ASC';
+    sortHeader = target;
+  } else {
+    sortOrder = 'DESC';
+    sortHeader = null;
+  }
+
+  const forSortedTable = [...body.children].sort((a, b) => {
+    const cellA = a.cells[toSort].innerText;
+    const cellB = b.cells[toSort].innerText;
+
+    if (sortOrder === 'ASC') {
+      return (cellA.includes('$')) === false
```

Comment:
`cellA.includes('$')` is already boolean, you don't need to compare it with `false`